types/address.go

```diff
@@ -26,6 +26,13 @@ func (a Address) String() string {
 	return base32.StdEncoding.WithPadding(base32.NoPadding).EncodeToString(checksumAddress)
 }
 
+var zeroAddr = [hashLenBytes]byte{}
```

Comment:
There's an issue open to make `ZeroAddr` a constant available to devs, so exporting this constant might knock that issue out too.